pkgs/applications/science/electronics/bitscope/common.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchurl, buildFHSUserEnv, makeWrapper, writeScriptBin, pkgs }:
```

Comment:
Please list each dependency as function parameter instead of taking the whole 'pkgs' set.